From cc4865221945809fc1f3205aa6d415cfa3b994db Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Fri, 27 May 2022 08:19:32 -0400
Subject: [PATCH] Do not force pickle5 in sys.path

---
 python/ray/__init__.py | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/python/ray/__init__.py b/python/ray/__init__.py
index 979ec207a..eaecfe673 100644
--- a/python/ray/__init__.py
+++ b/python/ray/__init__.py
@@ -57,13 +57,6 @@ def _configure_system():
         )
         os.environ["OMP_NUM_THREADS"] = "1"
 
-    # Add the directory containing pickle5 to the Python path so that we find
-    # the pickle5 version packaged with ray and not a pre-existing pickle5.
-    pickle5_path = os.path.join(
-        os.path.abspath(os.path.dirname(__file__)), "pickle5_files"
-    )
-    sys.path.insert(0, pickle5_path)
-
     # Importing psutil & setproctitle. Must be before ray._raylet is
     # initialized.
     thirdparty_files = os.path.join(
-- 
2.23.0

