From 518e9756bf74c26b8d1acf818b4693f99717e85a Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Fri, 6 Jan 2023 01:11:43 -0500
Subject: [PATCH] Do not force pickle5 in sys.path

---
 python/ray/__init__.py | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/python/ray/__init__.py b/python/ray/__init__.py
index 8aa0247dd..9efbd7dfd 100644
--- a/python/ray/__init__.py
+++ b/python/ray/__init__.py
@@ -45,15 +45,6 @@ def _configure_system():
     # MUST add pickle5 to the import path because it will be imported by some
     # raylet modules.
     #
-    # When running Python version < 3.8, Ray needs to use pickle5 instead of
-    # Python's built-in pickle. Add the directory containing pickle5 to the
-    # Python path so that we find the pickle5 version packaged with Ray and
-    # not a pre-existing pickle5.
-    if sys.version_info < (3, 8):
-        pickle5_path = os.path.join(
-            os.path.abspath(os.path.dirname(__file__)), "pickle5_files"
-        )
-        sys.path.insert(0, pickle5_path)
 
     # Check that grpc can actually be imported on Apple Silicon. Some package
     # managers (such as `pip`) can't properly install the grpcio library yet,
-- 
2.23.0

