From 5fb27cd52e0ee1fe28c3104c7c6419cbfc88092c Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Fri, 6 Jan 2023 02:12:40 -0500
Subject: [PATCH 1/4] Disable making non-core entry scripts

---
 python/setup.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 0773027a3..e7d838a0c 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -780,9 +780,9 @@ setuptools.setup(
     entry_points={
         "console_scripts": [
             "ray=ray.scripts.scripts:main",
-            "rllib=ray.rllib.scripts:cli [rllib]",
-            "tune=ray.tune.cli.scripts:cli",
-            "serve=ray.serve.scripts:cli",
+            #"rllib=ray.rllib.scripts:cli [rllib]",
+            #"tune=ray.tune.cli.scripts:cli",
+            #"serve=ray.serve.scripts:cli",
         ]
     },
     package_data={
-- 
2.23.0

