From 25140177653cc6714fec50496fc7d0d77a2441c3 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Mon, 13 Jun 2022 02:32:38 -0400
Subject: [PATCH] Remove all dependencies from setup.py

---
 python/setup.py | 62 +++++++------------------------------------------
 1 file changed, 9 insertions(+), 53 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index c6a7c263d..5d74b742a 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -200,30 +200,12 @@ ray_files += [
 # in this directory
 if setup_spec.type == SetupType.RAY:
     setup_spec.extras = {
-        "data": [
-            "pandas",
-            "pyarrow >= 6.0.1, < 7.0.0",
-            "fsspec",
-        ],
-        "default": [
-            "aiohttp >= 3.7",
-            "aiohttp_cors",
-            "colorful",
-            "py-spy >= 0.2.0",
-            "requests",
-            "gpustat >= 1.0.0b1",  # for windows
-            "opencensus",
-            "prometheus_client >= 0.7.1, < 0.14.0",
-            "smart_open",
-        ],
-        "serve": ["uvicorn==0.16.0", "requests", "starlette", "fastapi", "aiorwlock"],
-        "tune": ["pandas", "tabulate", "tensorboardX>=1.9", "requests"],
-        "k8s": ["kubernetes", "urllib3"],
-        "observability": [
-            "opentelemetry-api==1.1.0",
-            "opentelemetry-sdk==1.1.0",
-            "opentelemetry-exporter-otlp==1.1.0",
-        ],
+        "data": [],
+        "default": [],
+        "serve": [],
+        "tune": [],
+        "k8s": [],
+        "observability": [],
     }
 
     if sys.version_info >= (3, 7):
@@ -243,17 +225,7 @@ if setup_spec.type == SetupType.RAY:
     if sys.version_info >= (3, 7, 0):
         setup_spec.extras["k8s"].append("kopf")
 
-    setup_spec.extras["rllib"] = setup_spec.extras["tune"] + [
-        "dm_tree",
-        "gym<0.22",
-        "lz4",
-        # matplotlib (dependency of scikit-image) 3.4.3 breaks docker build
-        # Todo: Remove this when safe?
-        "matplotlib!=3.4.3",
-        "scikit-image",
-        "pyyaml",
-        "scipy",
-    ]
+    setup_spec.extras["rllib"] = setup_spec.extras["tune"] + []
 
     setup_spec.extras["all"] = list(
         set(chain.from_iterable(setup_spec.extras.values()))
@@ -263,23 +235,7 @@ if setup_spec.type == SetupType.RAY:
 # should be carefully curated. If you change it, please reflect
 # the change in the matching section of requirements/requirements.txt
 if setup_spec.type == SetupType.RAY:
-    setup_spec.install_requires = [
-        "attrs",
-        "click >= 7.0, <= 8.0.4",
-        "dataclasses; python_version < '3.7'",
-        "filelock",
-        "grpcio >= 1.28.1, <= 1.43.0",
-        "jsonschema",
-        "msgpack >= 1.0.0, < 2.0.0",
-        "numpy >= 1.16; python_version < '3.9'",
-        "numpy >= 1.19.3; python_version >= '3.9'",
-        "protobuf >= 3.15.3, < 4.0.0",
-        "pyyaml",
-        "aiosignal",
-        "frozenlist",
-        "requests",
-        "virtualenv",  # For pip runtime env.
-    ]
+    setup_spec.install_requires = []
 
 
 def is_native_windows_or_msys():
@@ -732,7 +688,7 @@ setuptools.setup(
     # The BinaryDistribution argument triggers build_ext.
     distclass=BinaryDistribution,
     install_requires=setup_spec.install_requires,
-    setup_requires=["cython >= 0.29.26", "wheel"],
+    setup_requires=[],
     extras_require=setup_spec.extras,
     entry_points={
         "console_scripts": [
-- 
2.23.0

