From 2aa949a3877b5f2e30a463e05cec067da13b18b1 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Fri, 9 Jun 2023 06:26:12 +0000
Subject: [PATCH] Ignore warnings in event.cc and logging.cc

---
 .bazelrc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/.bazelrc b/.bazelrc
index e81c646c15..892bfc6a19 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -44,6 +44,10 @@ build:clang-cl --per_file_copt="-\\.(asm|S)$@-Werror"
 build:msvc-cl     --per_file_copt="-\\.(asm|S)$@-WX"
 # Ignore warnings for protobuf generated files and external projects.
 build --per_file_copt="\\.pb\\.cc$@-w"
+# Ignore one specific warning
+build --per_file_copt="event\\.cc$@-w"
+build --per_file_copt="logging\\.cc$@-w"
+
 build:linux --per_file_copt="-\\.(asm|S)$,external/.*@-w,-Wno-error=implicit-function-declaration"
 build:macos --per_file_copt="-\\.(asm|S)$,external/.*@-w,-Wno-error=implicit-function-declaration"
 # Ignore minor warnings for host tools, which we generally can't control
-- 
2.34.1

