From d2851dfa4d2c32aaef7218107fbe621ccb0cf8b9 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Fri, 6 Jan 2023 02:20:11 -0500
Subject: [PATCH 3/4] Ignore warnings in event.cc and logging.cc

---
 .bazelrc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.bazelrc b/.bazelrc
index 1c4b9f296..bf429439a 100644
--- a/.bazelrc
+++ b/.bazelrc
@@ -41,6 +41,9 @@ build:clang-cl --per_file_copt="-\\.(asm|S)$@-Werror"
 build:msvc-cl     --per_file_copt="-\\.(asm|S)$@-WX"
 # Ignore warnings for protobuf generated files and external projects.
 build --per_file_copt="\\.pb\\.cc$@-w"
+# Ignore one specific warning
+build --per_file_copt="event\\.cc$@-w"
+build --per_file_copt="logging\\.cc$@-w"
 build --per_file_copt="-\\.(asm|S)$,external/.*@-w"
 #build --per_file_copt="external/.*@-Wno-unused-result"
 # Ignore minor warnings for host tools, which we generally can't control
-- 
2.23.0

