From 59888d5c6a2978770d0754082bdb71f738cba3e8 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Fri, 6 Jan 2023 02:21:06 -0500
Subject: [PATCH 4/4] Add bazel linkopts/libs

---
 python/setup.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/python/setup.py b/python/setup.py
index 655f92f40..05a284a21 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -444,6 +444,8 @@ def build(build_python, build_java, build_cpp):
         raise OSError(msg)
 
     bazel_env = dict(os.environ, PYTHON3_BIN_PATH=sys.executable)
+    bazel_env.setdefault("BAZEL_LINKOPTS", "-static-libstdc++:-lm")
+    bazel_env.setdefault("BAZEL_LINKLIBS", "-l%:libstdc++.a:-l%:libstdc++fs.a")
 
     if is_native_windows_or_msys():
         SHELL = bazel_env.get("SHELL")
-- 
2.23.0

