From 84887a38693e8aca83ca83a7dd75de75414dc0e0 Mon Sep 17 00:00:00 2001
From: Deepali Chourasia <deepch23@in.ibm.com>
Date: Fri, 25 Aug 2023 13:23:42 +0000
Subject: [PATCH] Changed min bazel version to 5.3

---
 WORKSPACE | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/WORKSPACE b/WORKSPACE
index f995b3b422..e7261cea05 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -22,7 +22,7 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 # it in CI.
 
 # Please keep this in sync with the .bazeliskrc file.
-versions.check(minimum_bazel_version = "5.4.1")
+versions.check(minimum_bazel_version = "5.3.0")
 
 # Tools to generate `compile_commands.json` to enable awesome tooling of the C language family.
 # Just run `bazel run @hedron_compile_commands//:refresh_all`
-- 
2.34.1

