From 7dc947d815641f190f8e93553b61a431fc46f798 Mon Sep 17 00:00:00 2001
From: Nishidha Panpaliya <npanpa23@in.ibm.com>
Date: Thu, 27 Apr 2023 11:41:53 -0400
Subject: [PATCH] Changed min bazel version to 5.3

---
 WORKSPACE       | 2 +-
 python/setup.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/WORKSPACE b/WORKSPACE
index 9300a57ed1..f5d95a704c 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -22,7 +22,7 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 
 # When the bazel version is updated, make sure to update it
 # in setup.py as well.
-versions.check(minimum_bazel_version = "5.4.0")
+versions.check(minimum_bazel_version = "5.3.0")
 
 # Tools to generate `compile_commands.json` to enable awesome tooling of the C language family.
 # Just run `bazel run @hedron_compile_commands//:refresh_all`
diff --git a/python/setup.py b/python/setup.py
index 32a93d01ba..dd011b5b8e 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -33,7 +33,7 @@ SUPPORTED_PYTHONS = [(3, 6), (3, 7), (3, 8), (3, 9), (3, 10), (3, 11)]
 # When the bazel version is updated, make sure to update it
 # in WORKSPACE file as well.
 
-SUPPORTED_BAZEL = (5, 4, 0)
+SUPPORTED_BAZEL = (5, 3, 0)
 
 ROOT_DIR = os.path.dirname(__file__)
 BUILD_JAVA = os.getenv("RAY_INSTALL_JAVA") == "1"
-- 
2.31.1

