From de6dd93b9f9d5051da551c321dc4ee734964acad Mon Sep 17 00:00:00 2001
From: Deepali Chourasia <deepch23@in.ibm.com>
Date: Fri, 28 Jul 2023 03:59:26 -0500
Subject: [PATCH] allow builds with root user

---
 WORKSPACE | 1 +
 1 file changed, 1 insertion(+)

diff --git a/WORKSPACE b/WORKSPACE
index eb6aeba907..e9aac6ba14 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -44,6 +44,7 @@ python_register_toolchains(
     name = "python3_9",
     python_version = "3.9",
     register_toolchains = False,
+    ignore_root_user_error = True,
 )
 
 load("@python3_9//:defs.bzl", bk_python = "interpreter")
-- 
2.23.0

